src/main/java/com/kodenigaz/desertwind/controller/UserController.java

```diff
@@ -0,0 +1,29 @@
+package com.kodenigaz.desertwind.controller;
+
+import java.security.Principal;
+import java.util.Base64;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.kodenigaz.desertwind.User;
+
+@RestController
+@CrossOrigin
+public class UserController {
+
+  @RequestMapping("/login")
+  public boolean login(@RequestBody User user) {
+    return user.getUserName().equals("user") && user.getPassword().equals("password");
```

Comment:
use private static final strings for magic text like "user", "password", "Authorization" etc.